app/models/manageiq/providers/cloud_manager/orchestration_template_runner.rb

```diff
@@ -1,5 +1,5 @@
 class ManageIQ::Providers::CloudManager::OrchestrationTemplateRunner < ::Job
-  DEFAULT_EXECUTION_TTL = 10 # minutes
+  DEFAULT_EXECUTION_TTL = 100 # minutes
```

Comment:
Can you change this constant name to include minutes, perhaps `DEFAULT_EXECUTION_TTL_MINUTES`?  We generally try to store the base unit of measure: seconds, bytes, etc.  I would not expect this to be in minutes.